<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Ignored" vsizetype="Ignored">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <widget class="QLabel" name="background_image">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>37</width>
      <height>12</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QStackedWidget" name="main_screen">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1000</width>
      <height>563</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>1000</width>
      <height>563</height>
     </size>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="Start">
     <widget class="QPushButton" name="start_button">
      <property name="geometry">
       <rect>
        <x>410</x>
        <y>440</y>
        <width>131</width>
        <height>61</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: #f8da27;
border: 4px solid white;
border-radius: 6px;
font-size: 24px;</string>
      </property>
      <property name="text">
       <string>START</string>
      </property>
      <property name="flat">
       <bool>false</bool>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>-10</y>
        <width>641</width>
        <height>431</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font-size: 120px;
color: white;</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap resource="resource.qrc">:/assets/assets/UNO_Logo.svg</pixmap>
      </property>
     </widget>
     <widget class="QLabel" name="server_error_label">
      <property name="geometry">
       <rect>
        <x>310</x>
        <y>510</y>
        <width>331</width>
        <height>51</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">font-size: 18px;
background-color: white;
color: black;</string>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="Game">
     <widget class="QLabel" name="card_deck_4">
      <property name="geometry">
       <rect>
        <x>600</x>
        <y>150</y>
        <width>131</width>
        <height>191</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap resource="resource.qrc">:/assets/assets/back.png</pixmap>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="card_deck">
      <property name="geometry">
       <rect>
        <x>570</x>
        <y>150</y>
        <width>131</width>
        <height>191</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap resource="resource.qrc">:/assets/assets/back.png</pixmap>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="card_deck_2">
      <property name="geometry">
       <rect>
        <x>580</x>
        <y>150</y>
        <width>131</width>
        <height>191</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap resource="resource.qrc">:/assets/assets/back.png</pixmap>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLabel" name="card_deck_3">
      <property name="geometry">
       <rect>
        <x>590</x>
        <y>150</y>
        <width>131</width>
        <height>191</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap resource="resource.qrc">:/assets/assets/back.png</pixmap>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="card_stack_button">
      <property name="geometry">
       <rect>
        <x>570</x>
        <y>150</y>
        <width>161</width>
        <height>191</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="flat">
       <bool>true</bool>
      </property>
     </widget>
     <zorder>card_deck</zorder>
     <zorder>card_deck_2</zorder>
     <zorder>card_deck_3</zorder>
     <zorder>card_deck_4</zorder>
     <zorder>card_stack_button</zorder>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="resource.qrc"/>
 </resources>
 <connections/>
</ui>
